ThirdPartyAdapters/vungle/vungle/src/main/java/com/vungle/mediation/VungleInterstitialAdapter.java

```diff
@@ -267,16 +278,26 @@ public void requestBannerAd(Context context,
     }
 
     private VungleListener mVungleBannerListener = new VungleListener() {
+
         @Override
-        void onAdEnd(String placement, boolean wasSuccessfulView, boolean wasCallToActionClicked) {
+        void onAdClick(String placementId) {
             if (mMediationBannerListener != null) {
-                if (wasCallToActionClicked) {
-                    // Only the call to action button is clickable for Vungle ads. So the
-                    // wasCallToActionClicked can be used for tracking clicks.
-                    mMediationBannerListener.onAdClicked(VungleInterstitialAdapter.this);
-                    mMediationBannerListener.onAdOpened(VungleInterstitialAdapter.this);
-                    mMediationBannerListener.onAdClosed(VungleInterstitialAdapter.this);
-                }
+                // Only the call to action button is clickable for Vungle ads.
+                mMediationBannerListener.onAdClicked(VungleInterstitialAdapter.this);
+                mMediationBannerListener.onAdOpened(VungleInterstitialAdapter.this);
+                mMediationBannerListener.onAdClosed(VungleInterstitialAdapter.this);

```

Comment:
shouldn't it be in onAdEnd ?

Reply:
From AdMob document, my understanding adClose() is different from adEnd():
```
onAdClosed() | When a user returns to the app after viewing an ad's destination URL, this method is invoked.

@Override
    public void onAdClosed() {
        // Code to be executed when the user is about to return
        // to the app after tapping on an ad.
    }
```

Reply:
yes, the description is bit different, we might need onAdClosed from sdk side as opposite event to onAdLeftApplication